crates/snark-acct-types/src/update.rs

```diff
@@ -75,6 +75,69 @@ impl UpdateOperationData {
     }
 }
 
+/// Subset of UpdateOperationData fields needed for unconditional update.
+/// This represents all the data of an update operation that is available to a DA-only synced OL
+/// fullnode.
+#[derive(Clone, Debug)]
+pub struct UpdateOperationUnconditionalData {
```

Comment:
So this is an `UpdateOperationData`.
without:

* `ledger_refs: LedgerRefs`
* `outputs: UpdateOutputs`

I haven't found any mention of `UpdateOperationUnconditionalData` in the specs, I think we need to update the specs.